clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/cache/LoggingInstrumentation.groovy

```diff
@@ -33,7 +35,8 @@ class LoggingInstrumentation implements ExecutionInstrumentation {
 
   @Override
   void executionCompleted(Agent agent) {
-    logger.info("${agent.providerName}:${agent.agentType} completed")
+    def durationMs = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - MetricInstrumentation.getAgentStartTimeNs(agent))
```

Comment:
Ya, I haven't seen it yet run with a cleared ThreadLocal but that could just be happenstance locally.

I'm just gassing that ThreadLocal, seems like the sensible thing to do.